R/ImpactFeatureWrapper.R

```diff
@@ -7,48 +7,74 @@
 #' @param learner [\code{\link{Learner}} | \code{character(1)}]\cr
 #'   The learner.
 #'   If you pass a string the learner will be created via \code{\link{makeLearner}}.
+#' @param slope.param [\code{integer}]\cr
+#'   Controls the rate of transition rate \eqn{\lambda}. Default is \code{100L}.
+#' @param trust.param [\code{integer}]\cr
+#'   Determines half of the minimal sample size for which we completely "trust" the conditional 
+#'   probability of a factor level. Default is \code{100L}
 #' @inheritParams createImpactFeatures
 #' @return [\code{\link{Learner}}].
 #' @export
-makeImpactFeaturesWrapper = function(learner, cols = NULL, fun = NULL) {
+makeImpactFeaturesWrapper = function(learner, cols = NULL, fun = NULL, slope.param = 100L, trust.param = 100L) {
   learner = mlr:::checkLearner(learner)
+  
   args = list(cols = cols, fun = fun)
+  
+  
+  if (!missing(slope.param) & !missing(trust.param)) {
```

Comment:
I agree. I was wondering if there's any chance that this could happen when using the ````makeWeightedClassesWrapper```` as framework, where the same occurs for the ````wcw.weight```` param.